x-pack/solutions/security/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/overview_tab.tsx

```diff
@@ -267,12 +268,22 @@ const getEvidenceList = ({ result }: CspFinding) =>
             />
           </EuiText>
           <EuiSpacer size={'s'} />
-          <CodeBlock language="json">{JSON.stringify(result?.evidence, null, 2)}</CodeBlock>
+          <CodeBlock language="json">
+            {JSON.stringify(getEvaluationEvidence(finding), null, 2)}
+          </CodeBlock>
         </>
       ),
     },
   ].filter(truthy);
 
+const getEvaluationEvidence = ({ result, resource }: CspFinding) => {
+  return (
+    result.evidence || // for all CIS benchmarks except CIS_GCP
+    (resource.raw as { resource?: unknown })?.resource || // for CIS_GCP benchmarks, prefer the specific resource
+    resource.raw // fallback to the raw resource
```

Comment:
TODO: 
1. fix property access as `resource.raw` will not be available for GCP wrapper types
2. use `getEvaluationEvidence` in `hasEvidence` ? 
3. add test